sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -125,16 +125,15 @@ def test_new_logic(LCClient):
 @pytest.mark.remote_data
 @pytest.mark.parametrize(
     "time, instrument",
-    [(a.Time("2012/10/4", "2012/10/6"), a.Instrument("goes")),
-     (a.Time('2013/10/5', '2013/10/7'), a.Instrument("goes"))])
+    [(a.Time("2012/10/4", "2012/10/5"), a.Instrument("goes"))])
 def test_fido(time, instrument):
-    qr = Fido.search(a.Time('2012/10/4', '2012/10/6'), Instrument('XRS'))
+    qr = Fido.search(time, Instrument('XRS'))
     assert isinstance(qr, UnifiedResponse)
     response = Fido.fetch(qr)
     assert len(response) == qr._numfile
 
 
-@settings(deadline=10000, max_examples=10)
+@settings(deadline=10000, max_examples=1)
```

Comment:
If we are only running it once is there any point? :stuck_out_tongue: 
```suggestion
@settings(deadline=10000, max_examples=5)
```

Reply:
Fixed.